collect_app/src/main/java/org/odk/collect/android/widgets/WidgetFactory.java

```diff
@@ -48,7 +48,7 @@ public static QuestionWidget createWidgetFromPrompt(FormEntryPrompt fep, Context
         // for now, all appearance tags are in english...
         appearance = appearance.toLowerCase(Locale.ENGLISH);
 
-        QuestionWidget questionWidget = new StringWidget(context, fep, readOnlyOverride);
+        final QuestionWidget questionWidget;
```

Comment:
I'm confused about the difference here. Does the scope not end up being the same?

Reply:
The trouble is that if it is not used, the one created here is simply garbage collected without `release` being called on its media player.

Reply:
Yep. Just realized that as I was looking at it again. 😅 Thanks!